runtime/src/iree/builtins/ukernel/BUILD

```diff
@@ -17,13 +17,15 @@ iree_runtime_cc_library(
     srcs = [
         "elementwise_generic.c",
         "elementwise_impl.c.inc",
-        "mmt4d_arm_64.c",
+        "mmt4d.c",
+        "mmt4d_arm64.c",
```

Comment:
we should keep this as arm_64 - we use that in other places and the intent is that it's the suffix of the IREE_UKERNEL_ARCH_ARM_64/IREE_ARCH_ARM_64 and consistent with what IREE_ARCH returns ("arm_64") so that we can programmatically include files if needed

Reply:
done